src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -601,6 +601,27 @@ public ExamResource(UserService userService, CourseService courseService, ExamSe
         return ResponseEntity.ok().body(notFoundStudentsDtos);
     }
 
+    /**
+     * POST /courses/:courseId/exams/:examId/addAllStudentsOfCourse : Add all users which are enrolled in the course to the exam so that the student can access the exam
+     *
+     * @param courseId     the id of the course
+     * @param examId       the id of the exam
+     * @return empty ResponseEntity with status 200 (OK) or with status 404 (Not Found)
+    */
+    @PostMapping(value = "/courses/{courseId}/exams/{examId}/addAllStudentsOfCourse")
```

Comment:
I don't really like the last part `addAllStudentsOfCourse`
I guess `register-course-students` might be better

Reply:
```suggestion
    @PostMapping(value = "/courses/{courseId}/exams/{examId}/register-course-students")
```

Reply:
done!